wesnoth-lib.po

```diff
@@ -1826,12 +1787,12 @@ msgid ""
 msgstr ""
 "<italic>text='Houbové háje'</italic> jsou rozlehlé podzemní porosty obřích "
 "hub, kterým se ve vlhké temnotě dobře daří. Většina jednotek má problémy s "
-"chůzi po měkkém povrchu nejmenších plodnic, ale na druhou stranu vzrostlejší "
-"kousky poskytují dobrý úkryt. Jízda ke své smůle nemá potřebnou volnost "
-"pohybu k boji. Nemrtví mají přirozenou zálibu v rozkladu a proto mají "
-"\"houbové háje v oblibě.\n"
+"chůzi po měkkém povrchu nejmenších plodnic, ale na druhou stranu jim "
+"vzrostlejší kousky poskytují dobrý úkryt. Jízda se zde hodně trápí, protože "
+"tu nemá potřebnou volnost pohybu k boji. Nemrtví mají přirozenou zálibu v "
+"rozkladu a proto mají houbové háje v oblibě.\n"
```

Comment:
přidávám čárku, a když už jsem v tom, ruším to druhé líbení (zálibu/oblibě)
Drough tam má obranu 60 %, tak snad ten můj návrh sedí
```suggestion
"rozkladu, a proto jsou v houbových hájích jako doma.\n"
```